requirements.txt

```diff
@@ -1 +1 @@
-pymongo==3.9.0
```

Comment:
You haven't removed the 3.9.0 wheels just audited them

Reply:
done